magni_bringup/config/default_robot.yaml

```diff
@@ -1,34 +1,69 @@
 
-# Robot Configuration
+# Default Robot Configuration
 
+# WARNING: Changing the content of this folder effects the working of magni_bringup/scripts/launch_core.py !!!
+# Make sure you understand what you're doing before changing anything in here. 
+
+
+# ============================================= USER CHANGEABLE PARAMETERS =================================================
 
 # Check for lidar and camera extrinsics files in two places with following priorities:
 #  1.) in ~/.ros/extrinsics/<SENSOR>_extrinsics_<POSITION>.yaml
 #  2.) in package magni_description/extrinsics/<SENSOR>_extrinsics_<POSITION>.yaml
-# where <SENSOR> is either "camera" or "lidar" and <POSITION> is an arbitrary string with which the extrinsics file is identified 
-raspicam: {'camera_installed' : 'True', 'position' : 'forward'}
-lidar: {'lidar_installed' : 'True', 'position' : 'top_plate'}
+# where <SENSOR> is either "camera" or "lidar" and <POSITION> is an arbitrary string with which the extrinsics file is identified
+raspicam_position: 'forward' # to disable insert "None"
+lidar_position : 'top_plate'  # to disable insert "None"
+
+sonars_installed: False # to enable set to True, to disable set to False
 
-sonars: None
-# sonars: 'pi_sonar_v1' # Use this to enable sonars
+force_time_sync : True
+oled_display: {'controller': None}
+
+# ============================================ ADMIN PARAMETERS FROM HERE DOWN ==============================================
 
-motor_controller:
-  board_version : None 
+ubiquity_motor:
+  board_version : None  # non ROS param - for launch_core.py to know which UR MCB board revision is present in the robot
   serial_port: '/dev/ttyAMA0'
   serial_baud: 38400
+  serial_loop_rate: 200
+  controller_loop_rate: 20
   pid_proportional: 5000
-  pid_integral: 7
+  pid_integral: 5
```

Comment:
Nitpik: This 'could' be as low as 3 but lets just go with this to not slow down this extremely long awaited change.  I checked kinetic-devel and noetic-devel and this is a mess so just leave this and we will do more evaluation and other things later.   Never hack some number at the 11th hour, it breaks stuff. We ship only what we have tested.